spec/faraday/request/authorization_spec.rb

```diff
@@ -63,6 +63,15 @@
       include_examples 'does not interfere with existing authentication'
     end
 
+    context 'when passed as callable' do
```

Comment:
Minor: this is about _a_ callable, right?

```suggestion
    context 'when passed a callable' do
```